### ISSUE TITLE
Can commitizen be pre-compiled?

### ISSUE THREAD

**Issue by jimthedev** [commitizen/cz-cli#39]
Right now we're using babel/register to run everything in commitizen and to get ES6 features. I'm doing this because I need to be able to require() adapters on the fly but still get the features of ES6. I'd love for most of the cli to be pre-built with babel, leaving only the adapter part to using babel/register. This would help git-cz to be faster. I've never done this before and am not even sure if it is possible. Any help would be great.

**Comment by tjoskar** [comment]
One can add:

```
"scripts": {
    "prepublish": "node node_modules/babel/bin/babel.js src --out-dir dist"
}
```

And then create a `.npmignore` file to ignore the source files to be publish:
```
/src
```

And point the `main` file in `package.json` to the `dist` folder:
```
"main": "dist/index.js"
```

I think that would do the trick.

I can create a pull request if you want.

**Comment by jimthedev** [comment]
If you want to put in a PR, that'd be excellent. My main concern is  that we do a lot of work with paths and I'd just need to find a way to test the built version. Thanks so much!

**Comment by jimthedev** [comment]
Closed by #53. Thanks @tjoskar!
